srv/salt/_modules/rgw.py

```diff
@@ -125,6 +125,16 @@ def create_bucket(**kwargs):
         return False
     return True
 
+def lookup_bucket(user, bucket):
+    s3conn = boto.connect_s3(
```

Comment:
I think this connection will fail because you are not passing the host/port properties and this function will be executing in the master_minion where no rgw daemon is running.
You should connect the same way as in the above `create_bucket` function.